### PR TITLE
EA archive Actions

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#61161](https://central.tri.be/issues/61161)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $link is never used | [Records.php:77](https://github.com/moderntribe/the-events-calendar/blob/48669f291561e35ab3c94c7d226a13362dd5058e/src/Tribe/Aggregator/Records.php#L77)
:warning: | The method parameter $context is never used | [Records.php:77](https://github.com/moderntribe/the-events-calendar/blob/48669f291561e35ab3c94c7d226a13362dd5058e/src/Tribe/Aggregator/Records.php#L77)
:warning: | The method parameter $link is never used | [Records.php:87](https://github.com/moderntribe/the-events-calendar/blob/48669f291561e35ab3c94c7d226a13362dd5058e/src/Tribe/Aggregator/Records.php#L87)
:warning: | The method parameter $context is never used | [Records.php:87](https://github.com/moderntribe/the-events-calendar/blob/48669f291561e35ab3c94c7d226a13362dd5058e/src/Tribe/Aggregator/Records.php#L87)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Record/List_Table.php [hunk 1]
```diff
@@ -303,6 +307,44 @@ public function get_columns() {
...
+		if ( current_user_can( $post_type_object->cap->edit_post, $post->ID ) ) {
+			$actions['edit'] = sprintf(
+				'<a href="%s">%s</a>',
+				get_edit_post_link( $post->ID ),
+				__( 'Edit', 'the-events-calendar' )
+			);
+
+			$actions['run-now'] = sprintf(
+				'<a href="%s">%s</a>',
+				get_edit_post_link( $post->ID ),
```
Comment (borkweb) [inline src/Tribe/Aggregator/Record/List_Table.php]: Should this one be a different URL?

Comment (bordoni) [inline src/Tribe/Aggregator/Record/List_Table.php]: Ooops, will fix it.

**Comment by bordoni** [comment]
Back in Code Review.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Edit.php:120](https://github.com/moderntribe/the-events-calendar/blob/21dceb0e78767402717045ced1018f2c4126f93d/src/Tribe/Aggregator/Tabs/Edit.php#L120)
:warning: | Comment refers to a TODO task | [Scheduled.php:108](https://github.com/moderntribe/the-events-calendar/blob/21dceb0e78767402717045ced1018f2c4126f93d/src/Tribe/Aggregator/Tabs/Scheduled.php#L108)
:x: | You must have a space after a closing parenthesis | [Scheduled.php:120](https://github.com/moderntribe/the-events-calendar/blob/21dceb0e78767402717045ced1018f2c4126f93d/src/Tribe/Aggregator/Tabs/Scheduled.php#L120)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Edit.php:120](https://github.com/moderntribe/the-events-calendar/blob/30360354c40e6b5c153aa9be93024482442e6145/src/Tribe/Aggregator/Tabs/Edit.php#L120)
:warning: | Comment refers to a TODO task | [Scheduled.php:108](https://github.com/moderntribe/the-events-calendar/blob/30360354c40e6b5c153aa9be93024482442e6145/src/Tribe/Aggregator/Tabs/Scheduled.php#L108)

> via jenkins/codesniffer-phpcs

**Comment by borkweb** [comment]
👘

**Pull request merged**